vendor/adminer/translations.php

```diff
@@ -0,0 +1,6 @@
+<?php
+				return array(
+					'openSite' => 'Open site',
+					'siteTables' => 'Site tables', 
+				);
```

Comment:
Is there a better way to do this? It's clunky, but works. I don't expect many other strings. Adminer has its own translations.